fury/tests/test_io.py

```diff
@@ -153,11 +154,35 @@ def test_pillow():
 
         for opt1, opt2 in [(True, True), (False, True), (True, False),
                            (False, False)]:
-            save_image(data, fname_path, use_pillow=opt1)
+            if not opt1:
+                with pytest.warns(UserWarning):
+                    save_image(data, fname_path, use_pillow=opt1)
+            else:
+                save_image(data, fname_path, use_pillow=opt1)
             data2 = load_image(fname_path, use_pillow=opt2)
             npt.assert_array_almost_equal(data, data2)
             npt.assert_equal(data.dtype, data2.dtype)
 
+        save_image(data, fname_path, use_pillow=True)
+        img_dpi = Image.open(fname_path).info.get('dpi')
+        assert abs(72 - img_dpi[0]) < 0.01
```

Comment:
Can this tolerance be defined as a constant in the test method?

Reply:
> Also there's a weird issue where the code coverage drops to 50% for no evident reason.

I have observed `codecov` reporting drops in the past that did not match the changes, so I'd not be worried in this case.

Please, squash all commits into a single one so that the commit history is kept cleaner/easier to navigate. I'd say that the PR would be ready to be merged then. Thanks @Nibba2018.

Reply:
Hello @jhlegarreta , I have squashed the commits.

Reply:
Done in d369613.